mergify_engine/web_cli.py

```diff
@@ -85,3 +85,16 @@ def queues():
             method="GET",
         )
     )
+
+
+def queues_v2():
```

Comment:
That's not being used though.
You might want to change the `queues` function directly then.